packages/suite-desktop/src/modules/coinjoin.ts

```diff
@@ -75,6 +78,13 @@ export const init: Module = ({ mainWindow }) => {
                     if (method === 'disable') {
                         logger.debug(SERVICE_NAME, `CoinjoinClient binary stop`);
                         coinjoinMiddleware.stop();
+                        powerSaveBlocker.stopBlockingPowerSave();
+                    }
+                    if (method === 'registerAccount') {
+                        powerSaveBlocker.startBlockingPowerSave();
+                    }
+                    if (method === 'unregisterAccount') {
+                        powerSaveBlocker.stopBlockingPowerSave();
```

Comment:
shouldn't this be also part of cleanup? somewhere inside dispose method?

Reply:
Yes, you are right, I added to dispose method. And rebased due to conflicts.